src/components/balance/ModalTransferAmount.vue

```diff
@@ -121,30 +132,32 @@
                       <input
                         class="w-full text-blue-900 dark:text-darkGray-100 text-2xl focus:outline-none bg-transparent placeholder-gray-300 dark:placeholder-darkGray-600"
                         inputmode="decimal"
+                        type="number"
                         min="0"
                         pattern="^[0-9]*(\.)?[0-9]*$"
                         placeholder="0.0"
-                        value=""
+                        v-model="transferAmt"
                       />
                     </div>
                     <button
                       type="button"
+                      @click="setMaxAmount(formatBalance)"
                       class="bg-blue-100 dark:bg-blue-200 hover:bg-blue-200 dark:hover:bg-blue-300 text-xs rounded-full px-3 py-2 text-blue-900 dark:text-darkGray-900 mx-3 focus:outline-none focus:ring focus:ring-blue-100 dark:focus:ring-blue-300"
                     >
                       MAX
                     </button>
                     <div
                       class="text-blue-900 dark:text-darkGray-100 text-lg border-l border-gray-300 dark:border-darkGray-500 px-3 py-4"
                     >
-                      XXX
+                      {{ unitToken }}
```

Comment:
When inputting the transfer value, the user should be able to choose a different unit of currency via a dropdown option (like choosing between ETH, Gwei, Wei, etc.)

Reply:
is that just conversion for formatting like PLD -> ETH, Gwei..? so should we transfer PLD to target address right?

Reply:
I mean adding a dropdown field with the various token denominators like this
![Screenshot from 2021-04-27 14-13-54](https://user-images.githubusercontent.com/40356749/116188745-e412c100-a762-11eb-85f7-97f39569f58c.png)